File: BasicMaths/AllDivisors/Divisors.py
import math
class Divisors:
    def findDivisors(self, number):
        divisors = []
        for i in range(1, int(number**0.5) + 1):
            if(number % i == 0):
                divisors.append(i)
                if(i != number // i):
                    divisors.append(number // i)
        
        divisors.sort()
        print(*divisors)
    
    def run(self):
        n = int(input("Enter the number: "))
        self.findDivisors(n)

divisors = Divisors()
divisors.run()

File: BasicMaths/AllDivisors/Divisors.ts
class Divisors {
  static findDivisors(n: number): void {
    let divisors: number[] = [];
    for (let i = 1; i <= Math.sqrt(n); i++) {
      if (n % i == 0) {
        divisors.push(i);

        if (i !== n / i) {
          divisors.push(n / i);
        }
      }
    }
    divisors.sort((a, b) => a - b);
    console.log(divisors.join(" "));
  }
}

Divisors.findDivisors(36);
